app/assets/stylesheets/application.scss

```diff
@@ -1,44 +1,68 @@
 @import "bootstrap";
 
+$orange: #f2580b;
+@mixin orange_background {
+  background-color: $orange;
+  border-color: $orange;
+}
+body {
+  background-color: #eee;
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.